src/client/theme-default/styles/components/vp-doc.css

```diff
@@ -109,13 +109,10 @@
 .vp-doc a {
   font-weight: 500;
   color: var(--vp-c-brand);
-  text-decoration-style: dotted;
-  transition: color 0.25s;
 }
 
 .vp-doc a:hover {
-  /* color: var(--vp-c-brand-dark); */
-  text-decoration: underline;
+  text-decoration: underline dotted;
```

Comment:
Just a suggestion, I think moving the dotted line down a bit looks a bit better. What's do your opinion?

```suggestion
  text-decoration: underline dotted;
  text-underline-offset: 2px;
```

![image](https://github.com/vuejs/vitepress/assets/31937175/0897a23b-5f52-490d-8ce9-a1c55b013056)
![image](https://github.com/vuejs/vitepress/assets/31937175/88d7a3ab-9e6e-4b0b-a0e5-4aadaceaf491)